lib/osml/model_endpoints/osml_endpoint_container.ts

```diff
@@ -69,7 +69,7 @@ export class OMSLEndpointContainer extends Construct {
    * @param {Construct} scope - The scope/stack in which to define this construct.
    * @param {string} id - The id of this construct within the current scope.
    * @param {OSMLEndpointContainerProps} props - The properties of this construct.
-   * @returns OMSLEndpointContainer - The OMSLEndpointContainer instance.
+   * @returns OSMLEndpointContainer - The OMSLEndpointContainer instance.
```

Comment:
Typo in the commend section 
- `The OMSLEndpointContainer instance.` should be  `The OSMLEndpointContainer instance.`